src/include/daos_task.h

```diff
@@ -298,6 +299,11 @@ typedef struct {
 	daos_prop_t		*prop;
 } daos_cont_set_prop_t;
 
+typedef struct {
```

Comment:
(style)  do not add new typedefs

Reply:
Ignoring for consistency with existing code